requirements.txt

```diff
@@ -8,3 +8,4 @@ scikit-learn>=0.19.2
 tqdm>=4.10.0
 dask[dataframe]>=2.9.0
 distributed>=2.11.0
+matrixprofile>=1.1.6
```

Comment:
Based on our versioning and the requirement of 1.1.7, this should be:

`matrixprofile>=1.1.7<2.0.0`

Reply:
Good catch. Is there a reason we're specifying version < 2.0.0?

Reply:
We use semantic versioning like most packages. A version less than 2 should guarantee compatibility. See

 https://link.medium.com/rIKVIvX34cb